mep/people/views.py

```diff
@@ -63,33 +63,42 @@ def get_form(self, *args, **kwargs):
         # initialize the form, caching on current instance
         if not self._form:
             self._form = super().get_form(*args, **kwargs)
-        # set minimum and maximum years for date range field
-            min_max = self.get_year_range()
-            if min_max:
-                self._form.set_membership_dates_placeholder(*min_max)
+        # set minimum and maximum years for date range fields
+            min_max_ranges = self.get_year_ranges('birth_year',
+                                                  'account_years')
+            if min_max_ranges:
+                self._form.set_daterange_placeholders(min_max_ranges)
```

Comment:
Why is this logic here and not in `get_form_kwargs` ? Can't everything be done at form initialization time?

Reply:
This is the logic that was previously there (in terms of when I passed it to form) for the membership date filters, if I recall.

It's pulled out separately because `get_year_ranges` lived on the view, which seemed more appropriate for something that was fetching information vs. setting that information on a form.

Reply:
I've refactored all of this to `get_form_kwargs`